src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -29,11 +29,74 @@ public boolean LIC0(List<Point> points, double length) throws IllegalArgumentExc
         return false;
     }
 
+    /**
+     * LIC #1 checks if there exists at least one set of three consecutive data points that cannot all be contained
+     * within or on a circle of the given <b>radius</b> (RADIUS1).
+     *
+     * @param points list of radar echos ({@link Point})
+     * @param radius has to be greater than 0 (zero)
+     * @return true if at least one set of three consecutive data points forms a circle with radius greater than
+     * given <b>radius</b>, false otherwise.
+     * @throws IllegalArgumentException is thrown if <b>radius</b> is less than 0 (zero)
+     */
+    public boolean LIC1(List<Point> points, double radius) throws IllegalArgumentException {
+        if (radius < 0) {
+            throw new IllegalArgumentException("Radius cannot be less than zero");
+        }
+        for (int i = 0; i < points.size() - 2; i++) {
+            Point point1 = points.get(i);
+            Point point2 = points.get(i + 1);
+            Point point3 = points.get(i + 2);
+            System.out.println("Radius: " + radiusOfSmallestCircle(point1, point2, point3));
```

Comment:
What do you think of not printing anyting to System.out since our program in the end should print YES (for launch) and NO (for no launch)? Do you think we can remove this print?

Reply:
Oh i forgot to remove this